test/index.js

```diff
@@ -1,7 +1,12 @@
 require('./basic')
+require('./cache-utils')
 require('./cache')
+require('./inflight-cache')
 require('./filters')
 require('./solc')
 require('./wallet')
 require('./subproviders/sanitizer')
 require('./subproviders/vm')
+// require('./subproviders/ipc')
+// require('./subproviders/etherscan')
```

Comment:
they were not recently being run as part of the test suite, and they didnt pass on one machine
( one required a running eth node, the other hits etherscan and didnt complete )